CMakeLists.txt

```diff
@@ -38,8 +38,12 @@ if (QUICKCPPLIB_REQUIRE_CXX17 OR QUICKCPPLIB_REQUIRE_CXX20)
 endif()
 # Make an interface only library so dependent CMakeLists can bring in this header-only library
 include(QuickCppLibMakeHeaderOnlyLibrary)
-# Make a docs target
-include(QuickCppLibMakeDoxygen)
+
+option(QUICKCPPLIB_ENABLE_DOXYGEN_DOCS_BUILD_SUPPORT "Enable doxygen docs build support" ON)
```

Comment:
I hate to be a pain :) However idiomatic practice in cmake is that `option`'s all go to the top of the cmake lists file, except of course when they can't because the cmake scripting language sucks. But in this case, it should be fine up top.

Here would be the right place:

```
cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
# If necessary bring in the quickcpplib cmake tooling
list(FIND CMAKE_MODULE_PATH "quickcpplib" quickcpplib_idx)
if(${quickcpplib_idx} GREATER -1)
  return()
endif()
## HERE!
set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmakelib")
```

Reply:
No worries, I've put it where you specified.